src/documents/RemoteModelAPI.md

```diff
@@ -0,0 +1,316 @@
+        </tr>
+    </tbody>
+</table>
+
+## Invokers
+
+### fetch
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                Initiates a fetch request. See the <a href="#fetch-workflow">Fetch Workflow</a> for more details.
+            </td>
+        </tr>
+        <tr>
+            <th>Default</th>
+            <td><code>"fluid.remoteModelComponent.fetch"</code></td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dt>that</dt>
+                <dd>
+                    The component.
+                </dd>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### fetchImpl
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                Must be supplied by an implementor to provide the concrete implementation for fetching data from the remote source. It must return a promise.
+            </td>
+        </tr>
+        <tr>
+            <th>Default</th>
+            <td><code>"fluid.notImplemented"</code></td>
+        </tr>
+    </tbody>
+</table>
+
+### write
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                Initiates a write request. See the <a href="#write-workflow">Write Workflow</a> for more details.
+            </td>
+        </tr>
+        <tr>
+            <th>Default</th>
+            <td><code>"fluid.remoteModelComponent.write"</code></td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dt>that</dt>
+                <dd>
+                    The component.
+                </dd>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+### writeImpl
+
+<table>
+    <tbody>
+        <tr>
+            <th>Description</th>
+            <td>
+                Must be supplied by an implementor to provide the concrete implementation for writing data to the remote source. It must return a promise that is resolved/rejected after the write operation has been handled by the remote end point.
+            </td>
+        </tr>
+        <tr>
+            <th>Default</th>
+            <td><code>"fluid.notImplemented"</code></td>
+        </tr>
+        <tr>
+            <th>Parameters</th>
+            <td>
+                <dt>payload</dt>
+                <dd>
+                    The data, from <code>that.model.local</code>, to send to the remote source.
+                </dd>
+            </td>
+        </tr>
+    </tbody>
+</table>
+
+## Fetch Workflow
+
+Executing the `fetch` invoker will add a fetch request and returns a promise for the result. Only one request can be in flight (processing) at a time. If a write request is in flight, the fetch will be queued. If a fetch request is already in queue/flight, the result of that request will be passed along to the current fetch request. When a fetch request is in flight , it will trigger the `fetchImpl` invoker to perform the actual request.
```

Comment:
See comment below in the similar wording for "the result of that request" for WRite